enhancements/windows-containers/health-management.md

```diff
@@ -221,6 +225,13 @@ validate that essential files have not been tampered with.
 }
 ```
 
+#### Permissions
+
+WICD needs a way to authenticate with the API server in order to pull down services ConfigMaps and watch node objects
+for a change in the desired version annotation. WICD will do this through a ServiceAccount. In order to limit WICD's
```

Comment:
I suggest making this one sentence along the lines of `WICD will use a new ServiceAccount to authenticate with the API server...`